packages/app-extensions/src/notification/README.md

```diff
@@ -42,21 +42,23 @@ Toaster action can be used to show notifications:
 
 `notification.toaster({key, type, title, body, icon, onClose, duration, time})`
 
-key: Unique identifier for removing a toaster 
-type:  One of`['neutral', 'success', 'warning', 'error', 'info']`
-title, body: Either a string, a html formatted string, a text resource key or any component
-icon: To overwrite default icons determined by type 
-onClose: Callback when toaster is closed. Parameter: Manually (Boolean) if it was closed by the user or by the duration 
-duration: Determines how long the toaster should be displayed in milliseconds. If -1 the toaster will not disappear automatically.
-time: Time as when the toaster action was executed. Is used to sort all toasters. Default is the add time.
+* key: Unique identifier for removing a toaster
+* type: One of`['neutral', 'success', 'warning', 'error', 'info']`
+* title: A string or text resource key
```

Comment:
The 2nd example contains a component as title. Is that also supported? Should we mention it here as well (similar to `body`?